RecoLocalFastTime/FTLClusterizer/src/MTDThresholdClusterizer.cc

```diff
@@ -237,6 +279,17 @@ FTLCluster MTDThresholdClusterizer::make_cluster(const FTLCluster::FTLHitPos& hi
               theTimeThreshold *
                   sqrt(theBuffer.time_error(r, c) * theBuffer.time_error(r, c) + seed_time_error * seed_time_error))
             continue;
+          if ((seed_subdet == GeomDetEnumerators::barrel) && (theBuffer.subDet(r, c) == GeomDetEnumerators::barrel)) {
+            double hit_error_xx = theBuffer.local_error(r, c).xx();
+            double hit_error_yy = theBuffer.local_error(r, c).yy();
+            std::cout << "thePositionThreshold: " << thePositionThreshold << std::endl;
```

Comment:
> You potentially added 3770 lines to the logs
cout is not allowed

Reply:
Hi,
I am so sorry that I forgot to delete that
Should I delete it and push the code now or wait for other comments?
Thanks.

Reply:
Please push the fix